src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+function averageAge(ages) {
+  return ages.reduce((previousAge, currentAge) => (
+    previousAge + currentAge), 0) / ages.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menFormPeople
+    = people
+      .filter(person => person.sex === 'm');
```

Comment:
not the best formating for the code, fix it in all places
```suggestion
  const menFormPeople = people.filter(person => (
    person.sex === 'm'
  ));
```

Reply:
thanks, fixed